db/crud_test.go

```diff
@@ -2282,3 +2282,97 @@ func TestSyncDataCVEqual(t *testing.T) {
 		})
 	}
 }
+
+func TestProposedRev(t *testing.T) {
+
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
```

Comment:
nitpicks:

I think this line is useful for debugging, but LevelDebug with KeyAll displays a lot of logging for rosmar that isn't helpful. When committing this, I think it's fine for this test to remove all the logging.